samples/CSharp/SourceGenerators/SourceGeneratorSamples/CsvGenerator.cs

```diff
@@ -7,7 +7,8 @@
 using Microsoft.CodeAnalysis.Text;
 using NotVisualBasic.FileIO;
 
-#nullable enable
+#nullable disable warnings
+#nullable enable annotations
```

Comment:
❓ Why are we disabling NRT here?

Reply:
I am also confused about why this change was made

Reply:
This was suggested by @terrajobst as something he thought we had agreed as standard for generators. I am not sure why. The code compiles and run either way.

Reply:
Reverted nullable change in ac5b4937. Please merge if happy so that I can submit the blog post.

Reply:
This one still needs  to be reverted.

Reply:
Apologies. Fixed in f65f6fb.